content/en/docs/reference/config/analysis/ist0134/index.md

```diff
@@ -0,0 +1,61 @@
+---
+title: ServiceEntryAddressesRequired
+layout: analysis-message
+owner: istio/wg-user-experience-maintainers
+test: no
+---
+
+This message occurs when ServiceEntry with protocol TCP (or unset) protocol doesn't have addresses defined in the manifest.
+
+## Example
+
+You will receive this message:
+
+{{< text plain >}}
+Warning [IST0134] (ServiceEntry service-entry.default serviceentry.yaml:13) ServiceEntry addresses are required for this protocol.
```

Comment:
Are `addresses` really _required_? The "How to resolve" section below makes it sound like it's not - it'll just bind all traffic to the port. This is a little confusing. @howardjohn

Reply:
Maybe describe the unexpected behavior that the use will see if they don't define addresses.

Reply:
@rvennam please check this, https://github.com/istio/istio/issues/27990#issuecomment-709593864

Reply:
Thanks. See my proposed suggestion below

Reply:
It is not required, I suggest we change this to "recommended"